packages/synchro-charts/src/testing/dynamicWidgetUtils/testCaseParameters.ts

```diff
@@ -9,6 +16,7 @@ export type SearchQueryParams = {
   width?: number | string;
   axis?: Axis.Options;
   height?: number | string;
+  size?: string;
```

Comment:
size in this format is an iot app kit concept, not synchro charts

Reply:
Done --- Change size type to `DialSizeConfig`